language/German/strings.po

```diff
@@ -1225,40 +1208,38 @@ msgid ""
 "\n"
 "If this is your first time using %2$s, please see the config.txt file in the "
 "docs directory for information on configuring %2$s."
-msgstr ""
+msgstr "Keine Maschinen gefunden. Bitte überprüfen Sie den Pfad für die ROMs in Ini-Datei %1$s.ini.\n\nWenn dies Ihre erste Verwendung von  %2$s ist, konsultieren Sie bitte config.txt im docs Verzeichnis zur Konfiguration von %2$s."
 
```

Comment:
Also, ich lese gelegentlich mal in einer Datei oder schaue hinein, aber "konsultieren"? Klingt mir ein bisschen fürnehm (natürlich eine Geschmacksfrage).